images/atomic-registry-quickstart/registry-console-template.yaml

```diff
@@ -0,0 +1,117 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: "registry-console-template"
+labels:
+  createdBy: "registry-console-template"
+parameters:
+  -
+    description: "The public url for the Openshift OAuth Provider"
+    name: OPENSHIFT_OAUTH_PROVIDER_URL
+    required: true
+  -
+    description: "The public url for the Openshift OAuth Provider"
+    name: COCKPIT_KUBE_URL
+    required: true
+  -
+    description: "The public url for the Openshift OAuth Provider"
+    name: COCKPIT_KUBE_INSECURE
+    required: false
+  -
+    description: "Oauth client secret"
+    name: OPENSHIFT_OAUTH_CLIENT_SECRET
+    from: "user[a-zA-Z0-9]{64}"
+    generate: expression
+  -
+    description: "Oauth client id"
+    name: OPENSHIFT_OAUTH_CLIENT_ID
+    value: "cockpit-oauth-client"
+  -
+    description: "Skip kubernetes CA verification"
+    name: KUBERNETES_INSECURE
+    value: ""
+  -
+    description: "PEM Encoded certificate to use for CA verification"
+    name: KUBERNETES_CA_DATA
+    value: ""
+objects:
+  -
+    kind: DeploymentConfig
+    apiVersion: v1
+    metadata:
+      name: "registry-console"
+    labels:
+      name: "registry-console"
+    spec:
+      replicas: 1
+      selector:
+      name: "registry-console"
+    template:
+      metadata:
+        labels:
+          name: "registry-console"
+      spec:
+        containers:
+          -
+            name: "registry-console"
+            image: "aweiteka/cockpit-registry:0.95"
+            ports:
+              -
+                containerPort: 9090
+                protocol: TCP
+            env:
+              -
+                name: OPENSHIFT_OAUTH_PROVIDER_URL
+                value: "${OPENSHIFT_OAUTH_PROVIDER_URL}"
+              -
+                name: OPENSHIFT_OAUTH_CLIENT_ID
+                value: "${OPENSHIFT_OAUTH_CLIENT_ID}"
+              -
+                name: KUBERNETES_INSECURE
+                value: "${KUBERNETES_INSECURE}"
+              -
+                name: KUBERNETES_CA_DATA
+                value: "${KUBERNETES_CA_DATA}"
+              -
+                name: COCKPIT_KUBE_INSECURE
+                value: "${COCKPIT_KUBE_INSECURE}"
```

Comment:
KUBERNETES_INSECURE tells cockpit to not validate the kubernetes API certificate before talking to it.

COCKPIT_KUBE_INSECURE tells the container to server cockpit without tls. http instead of https